src/app/Gallery/page.tsx

```diff
@@ -0,0 +1,157 @@
+import GalleryComponent from '@/components/ui/GalleryComponent';
+import type { Metadata } from 'next';
+import data from '@/data/sportGalleryData.json';
+import Image from 'next/image';
+
+export const metadata: Metadata = {
+  title: 'Gallery Page',
+  description: '',
+};
+
+const getRandom = (arr) => arr[Math.floor(Math.random() * arr.length)];
+
+export default function Gallery() {
+  const borderColors = ['pink', 'white', 'green'];
+
+  const titleColors = {
+    pink: ['pinkWhite', 'pinkGreen', 'pinkBlue'],
+    green: ['greenPink', 'greenBlue'],
+    white: ['whiteBlue', 'whitePink'],
+  };
+
+  const getRandomColorPair = () => {
+    const randomBorderColor = getRandom(borderColors);
+    const matchingTitleColors = titleColors[randomBorderColor];
+    const randomTitleColor = getRandom(matchingTitleColors);
+    return { borderColor: randomBorderColor, titleColor: randomTitleColor };
+  };
+
+  const positions = ['upperLeft', 'upperRight', 'lowerLeft', 'lowerRight'];
+
+  return (
+    <main className="bg-blue pt-20">
+      {/* Display the first item */}
+      <div>
+        <GalleryComponent
+          image={data[0].image}
+          sportName={data[0].title}
+          layout="wide"
+          borderColor="green"
+        />
+      </div>
+
+      {/* Gallery header */}
+      <div className="flex justify-center items-center pt-8 pb-24">
+        <h1 className="font-jersey10 text-green pr-4 sm:pr-6 md:pr-8 lg:pr-10 xl:pr-12 text-4xl sm:text-5xl md:text-6xl lg:text-7xl xl:text-8xl">
```

Comment:
- use ```px``` so that it will have padding for both left and right
- center text for sm screens
- add more padding on the sides for smaller screen
![image](https://github.com/user-attachments/assets/7880d1c2-d669-4f5e-b355-f59064d76e42)